motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 <xacro:macro name="motoman_motopos_d500" params="prefix">
-
+  <xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
```

Comment:
Doesn't look like this is used in this file.
Please change the visual tags to use the macros from the common materials file instead of using just a name.